frontend/src/app/createJournal.tsx

```diff
@@ -102,6 +120,32 @@ export default function CreateJournal() {
     // TODO error handling
     if (!firebaseUser) return;
     const token = await firebaseUser.getIdToken();
+
+    // Edit Mode
+    if (isEditMode && entryId) {
+      const updatedEntry = await updateJournalEntry({
```

Comment:
Can you add error handling here and for `createJournalEntry` to display an error message with `Alert.alert` if creating/updating fails? See other files for examples